cluster-up/cluster/images.sh

```diff
@@ -18,6 +18,7 @@ if [ -z $KUBEVIRTCI_PROVISION_CHECK ]; then
     IMAGES[okd-4.2]="okd-4.2@sha256:a830064ca7bf5c5c2f15df180f816534e669a9a038fef4919116d61eb33e84c5"
     IMAGES[okd-4.3]="okd-4.3@sha256:63abc3884002a615712dfac5f42785be864ea62006892bf8a086ccdbca8b3d38"
     IMAGES[ocp-4.3]="ocp-4.3@sha256:8f59d625852ef285d6ce3ddd6ebd3662707d2c0fab19772b61dd0aa0f6b41e5f"
```

Comment:
please update ocp-4.3 as well to the latest hash , 
find it at 
https://github.com/kubevirt/kubevirtci/pull/246/files

it was reverted by mistake
there is an issue about those two